TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/Workspace.php

```diff
@@ -560,7 +565,7 @@ protected function findNodeDataInTargetWorkspace(NodeInterface $node, Workspace
      * @return void
      * @Flow\Signal
      */
-    protected function emitBaseWorkspaceChanged(Workspace $workspace, Workspace $oldBaseWorkspace, Workspace $newBaseWorkspace)
+    protected function emitBaseWorkspaceChanged(Workspace $workspace, Workspace $oldBaseWorkspace = null, Workspace $newBaseWorkspace = null)
```

Comment:
The emitter signature has no direct effect on the slot, only when calling the emitter this makes the arguments optional now. No BC break. Totally.